### PR TITLE
Explore: Fix showing of Prometheus data in Query inspector

### PR DESCRIPTION
**Pull request by ivanahuckova**
**What this PR does / why we need it**:
When running Prometheus queries in Explore, the request and response wasn't shown in Inspector panel because if we have received response with `hideFromInspector === true` after, the result was overwritten. This PR fixes it. 

**Fixed:** 
after

**Before:**
before

### CODE REVIEW

**Review by zoltanbedi — COMMENTED** [review]
This looks good to me. Nice job on adding tests. Can you please try to convert it to use react-testing-library? I think we agreed that we want to migrate to that and write new tests with that. Also in the future it might make sense to unify this inspector component with the one we have in dashboard.

File: public/app/features/explore/ExploreQueryInspector.tsx [hunk 1]
```diff
@@ -57,15 +57,15 @@ function stripPropsFromResponse(response: any) {
   return clonedResponse;
 }
 
-interface Props {
+export interface Props {
```
Comment (Elfo404) [inline public/app/features/explore/ExploreQueryInspector.tsx:60]: It's really just a nit (_and personal and totally opinionanted, so it's even less important than a nit 😄  )_, but I personally would avoid exporting anything that is not really needed outside of a module as I feel it adds noise.

In this case, `Props` is not really needed outside of here apart from the test, in which case you can do: 

```ts
import React, { ComponentProps } 'from 'react';
import { ExploreQueryInspector } from './ExploreQueryInspector';

type ExploreQueryInspectorProps = ComponentProps<typeof ExploreQueryInspector>;
```

what do you think?

Comment (ivanahuckova) [reply public/app/features/explore/ExploreQueryInspector.tsx:60]: Uuu I like it and I agree! Will update this. Thanks!

**Review by zoltanbedi — APPROVED** [review]
Good job! 💿

**Review by Elfo404 — APPROVED** [review]
LGTM 🚀

**Pull request merged**